newIDE/app/src/EventsSheet/EventsTree/Renderers/JsCodeEvent.js

```diff
@@ -186,9 +187,11 @@ export default class JsCodeEvent extends React.Component<
         tabIndex={0}
         style={textStyle}
       >
-        {parameterObjects
-          ? `, objects /*${parameterObjects}*/`
-          : ' /* Click here to choose objects to pass to JavaScript */'}
+        {parameterObjects ? (
+          <Trans>{`, objects /*${parameterObjects}*/`}</Trans>
+        ) : (
+          <Trans>{`\u00A0/* Click here to choose objects to pass to JavaScript */`}</Trans>
```

Comment:
```
<>
{' '}
<Trans>/* Click here....*/</Trans>
</>
```

You might need to escape / and *.

Reply:
Or maybe:
```
<>
&nbsp;
<Trans>/* Click here....*/</Trans>
</>
```

Search how to escape / and * in React JSX (not sure if you need to, maybe not!)

Reply:
Not sure
![image](https://user-images.githubusercontent.com/1670670/103376001-c8ace780-4adb-11eb-940a-17976368ce35.png)